### PR TITLE
Update Windows docs networking.md

### PR DESCRIPTION
**Pull request by tzifudzi**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
---
**Is your idea request related to a problem that you've solved? Please describe.**
Yes. When using Windows nodes in an Amazon EKS cluster with the default secondary IPv4 address assignment mode, there are currently no supported configurations to manipulate how the VPC Resource Controller requests and allocates IPv4 addresses.

**Describe the best practice**
Given the known limitation and its potential limitations, this documentation adds some workarounds and brings awareness to the limitation.

**Describe alternatives you've considered**
Alternatives have been included in the documentation, namely,
- scheduling workloads in a way that optimizes number of available IPs 
- changing subnet configurations to mitigate potential impact

**Additional context**
- Amazon EKS documentation will also be updated to reference this documentation

### CODE REVIEW

File: content/windows/docs/networking.md [hunk 1]
```diff
@@ -71,6 +71,30 @@ DSR enablement can be verified following the instructions in the [Microsoft Netw
 
 ![](./images/dsr.png)
 
+If preserving your available IPv4 addresses and minimizing wastage is crucial for your subnet, it is generally
+recommended to avoid using prefix delegation mode as mentioned in
+[Prefix Mode for Windows - When to avoid](https://aws.github.io/aws-eks-best-practices/networking/prefix-mode/index_windows/#avoid-prefix-delegation-when).
+If using prefix delegation is still desired, you can take steps to optimize IPv4 address utilization in your subnet. The
+documentation Prefix Mode for 
+[Windows - Configure parameters](https://aws.github.io/aws-eks-best-practices/networking/prefix-mode/index_windows/#configure-parameters-for-prefix-delegation-to-conserve-ipv4-addresses)
```
Comment (geoffcline) [inline content/windows/docs/networking.md:79]: right now, the link style is pointing out to the internet

will you change the link style to reference the files in the repo?

I sent you a patch with an example.

**Review by geoffcline — CHANGES_REQUESTED** [review]
good with content, requested minor syntax change

File: content/windows/docs/networking.md (hunk 1 above)
Comment (tzifudzi) [reply content/windows/docs/networking.md]: @geoffcline Please take a look. The URLs have been updated as suggested.

**Comment by geoffcline** [comment]
great!

**Review by geoffcline — APPROVED** [review]

**Comment by geoffcline** [comment]
@svennam92 can we prioritize this so I can deploy some corresponding content in the EKS docs?

**Comment by svennam92** [comment]
@CodiumAI-Agent /review

**Comment by svennam92** [comment]
@CodiumAI-Agent /improve

**Comment by svennam92** [comment]
@tzifudzi Overall this PR looks good to me. Can you please review the suggestions from the AI agent above and also make sure you are not using line breaks within a paragraph — each paragraph should on the same line number.

**Comment by tzifudzi** [comment]
Thanks @svennam92 for the review. I applied the suggestion to remove line breaks so each paragraph is on its own line number. I also applied the suggestions from the AI assistant - really cool tool!

**Review by svennam92 — APPROVED** [review]
Thank you for the PR!

**Pull request merged**